usr/src/lib/brand/lx/zone/lx_boot_zone_busybox.ksh

```diff
@@ -36,11 +36,16 @@ depend() {
     keyword nojail noprefix novserver
 }
 start() {
+EOF
+# Only alter resolve.conf if we're getting info from zonecfg(1M).
+zonecfg -z $ZONENAME info attr name=resolvers | grep -q resolvers
+if [[ $? == 0 ]]; then
+    cat > $tmpfile <<EOF
     if [ ! -e /etc/resolv.conf ]; then
         echo "# AUTOMATIC ZONE CONFIG" > /etc/resolv.conf
-EOF
-zonecfg -z $ZONENAME info attr name=resolvers |
-awk '
+    EOF
```

Comment:
If you indent the EOF, it won't terminate the heredoc..
(unless that's a bash extension)

Reply:
Not sure, but I fixed it at our end.